FirebaseAuth/Sources/AuthProvider/OAuth/FIROAuthProvider.m

```diff
@@ -79,6 +79,12 @@ @implementation FIROAuthProvider {
      otherwise.
    */
   BOOL _usingClientIDScheme;
+
+  /** @var _customAuthDomain
+   @brief The custom authentication domain used to handle all sign-in redirects. End-users will see
+   this domain when signing in. This domain must be allowlisted in the Firebase Console.
+   */
+  NSString *_customAuthDomain;
```

Comment:
What is this value used for?

Reply:
Thanks for pointing it out! It was initially being used to store auth domain for OAuthProvider, but WebUtils handles this well. Fixed it.